app/pages/Recovery/PerformRecovery.tsx

```diff
@@ -33,7 +33,8 @@ async function getPrfKeySeed(
     identityNumber: number
 ) {
     setMessage('Please allow recovering credentials');
-    const prfKeySeed = await ledger.getPrfKeyRecovery(identityNumber);
+    // Version = 0, because Recovery always uses the seed, to check both versions.
```

Comment:
I do not understand this comment. If `getPrfKeyRecovery` always returns the seed, then why would it take the `version` as input?

Reply:
I guess it shouldn't take the version.